scripts/install.sh

```diff
@@ -11,20 +11,20 @@ if [ "$UNAME" != "Linux" ] && [ "$UNAME" != "Darwin" ] ; then
 fi
 
 if [ "$UNAME" = "Darwin" ] ; then
-  PLATFORM="macos"
+  FILENAME="spectral-macos"
 elif [ "$UNAME" = "Linux" ] ; then
-  PLATFORM="linux"
```

Comment:
Right now we have `spectral-macos` for macos and `spectral` for linux. This should be fixed properly another time but quick fix here is to point to the right file.